pyfifotax/utils.py

```diff
@@ -72,20 +72,30 @@ def get_reference_rates():
     return daily_ex_rates, monthly_ex_rates, supported_currencies
 
 
-def get_monthly_rate(monthly_rates: pd.DataFrame, date: datetime, currency: str):
+def get_monthly_rate(
+    monthly_rates: pd.DataFrame, date: datetime, currency: str, domestic_currency: str
+):
+    if currency == domestic_currency:
+        return to_decimal(1)
+
     return to_decimal(monthly_rates[currency][date.year, date.month])
 
 
-def get_daily_rate(daily_rates: pd.DataFrame, date: datetime, currency: str):
+def get_daily_rate(
+    daily_rates: pd.DataFrame, date: datetime, currency: str, domestic_currency: str
+):
+    if currency == domestic_currency:
```

Comment:
If `domestic_currency` is not EUR, this will be erroneous (because in the forex .csv the EUR column is missing).

I also thought it'd be better to handle it in a general way because it's possible that a user received some amount of money in a foreign currency. In this case, no conversion happens, that money just "appears" on the account.

Reply:
I'll get rid of this for now. It indeed makes no sense to have this flag if the exchange rate files assumes EUR to be there in the first place. 
Deposits of other currency are now possible if source_currency == target_currency. FIFO considerations then starts at the day of deposit.

Reply:
closing as there now is a dedicated movement tab